test/runtests.sh

```diff
@@ -7,7 +7,7 @@ then
 fi
 
 # delete output from previous run
-if [ ! -d "../actual.txt" ]
+if [ -e "./actual.txt" ]
```

Comment:
As an aside, this whole block is probably better served with a straightforward `rm -f actual.txt`.